pkg/api/server/config/config.go

```diff
@@ -40,7 +39,7 @@ func Get() *Config {
 	viper.SetConfigName("config")
 	viper.SetConfigType("env")
 	viper.AddConfigPath("/etc/tekton/results")
-	viper.AddConfigPath("config/env")
+	viper.AddConfigPath("config/base/env")
```

Comment:
I think this could break deployments - is this meant to facilitate contributor testing?

Reply:
Yes, that's right. Since the `config` directory was restructured into overlays in a previous PR, running the server directly from IDE was searching for the file in the `config/env` directory, which doesn't exist now.
This will NOT impact deployment, because deployment is using the absolute path `/etc/tekton/results` to mount config.
https://github.com/tektoncd/results/blob/2b03ffa8c753be7c07a67f9b6d78ac49c3370dba/config/base/api.yaml#L54